### ISSUE TITLE
Updated Junkyard?

### ISSUE THREAD

**Issue by antdude** [gfax/junkyard#1]
I just noticed Junkyard project is alive in https://github.com/gfax/junkyard/commits/98edac55e3cd70f19deb408a36547a8ff78d1164.atom ... New features or something?

**Comment by kfprimm** [comment]
Yeah, we're going to rewrite the core logic with a complete test suite. After that, the rbot plugin will be the first project to use the new library.

This means the game should become much more stable and will allow easier customization.

When you report a bug in the future, we'll be able to recreate it as a scenario (like [this](https://github.com/gfax/junkyard/blob/master/features/games/default.feature#L3-L12)). This will allow use to identify the bug and make sure that it never becomes a problem again.
